### PR TITLE
Storybook button

### PR DESCRIPTION
**Pull request by Caruki**
Add styled button component and matching story

See the Preview here:
https://show-cased-p-storybook--pdsbj2.herokuapp.com/storybook/index.html?path=/story/buttons--sign-up-button

### CODE REVIEW

File: client/src/components/SubmitButton.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const SubmitButton = styled.button`
+  width: 136px;
+  height: 46px;
+  border-radius: 20px;
+  background-image: linear-gradient(#504481, #1e194f);
+  color: #d05888;
+  font-size: 17px;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 300;
+
```
Comment (Susanne1307) [inline client/src/components/SubmitButton.js:12]: You could also think about adding `outline: none` to get rid of the blue border when clicking on the buttton :)

Comment (Caruki) [reply client/src/components/SubmitButton.js:12]: oh right. thank you for the advice! I forgot 😅

File: client/src/components/SubmitButton.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
+
+const SubmitButton = styled.button`
+  width: 136px;
```
Comment (Susanne1307) [inline client/src/components/SubmitButton.js:4]: Have a look at the comments regarding fixed widths on this PR: 
https://github.com/florianGierlichs/mycrypt/pull/24

> But I don't think you should set the height and width at all. It should be calculated automatically by the content and font-size.
If you have a button designed, take a look at the padding and font-size. This is all you need.
If you set a fixed width, the button would have issues with long content.

Comment (Caruki) [reply client/src/components/SubmitButton.js:4]: Thank you for the heads up! I will consider this in the other components. However this button only has two states with same size. So it should be fine for this component.

**Pull request merged**